caching-attribute-service-client/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  `java-library`
+  jacoco
+  id("org.hypertrace.jacoco-report-plugin")
```

Comment:
publish?

Reply:
Not being used yet, but might as well add it now I suppose.